values-it/strings.xml

```diff
@@ -1,142 +1,181 @@
+            <string name="navigation_drawer_close">Chiudi drawer navigazione</string>
             <string name="label_accept">Accetta</string>
             <string name="label_accepted">Accettato</string>
             <string name="label_decline">Rifiuta</string>
             <string name="label_delete">Elimina</string>
-            <string name="title_optional_app_update">Aggiornamento App disponibile</string>
-            <string name="ms_prev">Precedente</string>
-            <string name="ms_next">Successivo</string>
-            <string name="ms_end">Completa</string>
+            <string name="label_delete_warning">Eliminare %1$s?</string>
+            <string name="dont_show_again">Non visualizzare più</string>
+            <string name="title_optional_app_update">Disponibile aggiornamento app</string>
+            <string name="ms_prev">Indietro</string>
+            <string name="ms_next">Avanti</string>
+            <string name="ms_end">Fine</string>
+        <!--FEEDBACK-->
+            <string name="label_now">Ora</string>
+            <string name="label_later">Più tardi</string>
+            <string name="label_never">Mai</string>
+            <string name="label_rate">Vota</string>
+            <string name="title_rate_us">Valuta quest’app</string>
+           <string name="message_rate_us">Speriamo che quest’app ti piaccia. Ti chiediamo non più di un minuto per valutarla o darci il tuo feedback. Questo potrebbe aiutare altri utenti a scoprire quest’app!</string>
+            <string name="label_supportproject">Supporta</string> <!--verb-->
+            <string name="message_supportproject">Se ti piacciono i pacchetti Open GApps, considera per favore di supportare il progetto. Lo sviluppo dei pacchetti è completamente open source e tutti noi beneficiamo degli sforzi degli sviluppatori!</string>
         <!--EXPLANATION-->
             <string name="explanation_permission_denied">Per funzionare correttamente devi garantire a Open GApps l’accesso all’archivio esterno. Ha senso, no?</string>
-            <string name="explanation_adblock_detected">Adblocker rilevato!? Le pubblicità possono essere fastidiose, ma tengono vivi il progetto e i suoi sviluppatori! Per favore considera la possibilità di disabilitare il tuo adblocker oppure di donare</string>
+            <string name="explanation_adblock_detected">Rilevato Adblocker? Le pubblicità possono essere fastidiose, ma tengono vivi il progetto e i suoi sviluppatori! Considera la possibilità di disabilitare il tuo Adblocker oppure di donare</string>
+            <string name="title_webview_not_installed">Oops! Nessun provider WebView installato!?</string>
+            <string name="message_webview_not_installed">Sembra che sia stato disinstallato \"Stock WebView\", ma la ROM non supporta \"Google WebView\". Ri-flasha la ROM e chiedi agli sviluppatori della ROM di aggiungere il supporto per provider alternativi WebView.</string>
```

Comment:
Isn't it better "provider WebView alternativi" than "provider alternativi WebView"?